src/network/serverpackethandler.cpp

```diff
@@ -861,6 +861,18 @@ void Server::handleCommand_PlayerItem(NetworkPacket* pkt)
 		return;
 	}
 
+	Inventory *inventory = &player->inventory;
+	InventoryList *mainlist = inventory->getList("main");
+	u32 mainlist_size = (mainlist == nullptr) ? 0 : mainlist->getSize();
```

Comment:
```suggestion
	auto *mainlist = player->inventory.getList("main");
	u32 mainlist_size = mainlist ? mainlist->getSize() : 0;
```

Reply:
Applied changes. However, I wonder when should I use “auto” in place of exact declarations.